packages/compass/src/main/application.js

```diff
@@ -182,6 +185,17 @@ Application.prototype.setupUserDirectory = function() {
   }
 };
 
+Application.prototype.setupLogging = function() {
+  const home = app.getPath('home');
+  const appData = app.getPath('appData');
```

Comment:
Do you think it's okay to use those instead of what was here before? Here's how [electron docs](https://www.electronjs.org/docs/api/app#appgetpathname) describe what those paths resolve to:

> - `home` User's home directory.
> - `appData` Per-user application data directory, which by default points to:
>      `%APPDATA%` on Windows
>      `$XDG_CONFIG_HOME` or `~/.config` on Linux
>      `~/Library/Application Support` on macOS

Reply:
Electron doesn’t expose `%LOCALAPPDATA%`, does it

Reply:
The thing is, this is something that should rather go into `%LOCALAPPDATA%` than `%APPDATA%`, since it’s not something that makes sense to share across machines, and we also ideally would want these files to be next to the mongosh log files (if they exist)

Reply:
Ahhh, yeah, I missed this detail, sorry. Yeah, there is no `getPath('localAppData')` or something similar as far as I can see, so I'll change this line to use env vars as it was before, thanks for spotting it!

Reply:
```suggestion
  const home = app.getPath('home');
  const appData = process.env.LOCALAPPDATA || process.env.APPDATA;
```